php/ec/.storybook/main.js

```diff
@@ -12,11 +14,13 @@ const addons = [
   '@ecl-twig/storybook-addon-diff/src/register',
 ];
 
-const managerWebpack = async baseConfig => {
-  // Exclude node_modules
-  baseConfig.module.rules[0].exclude = /node_modules\/(?!@ecl-twig\/).*/;
-
-  return baseConfig;
+module.exports = {
+  stories,
+  addons,
+  webPackFinal: (config) => {
```

Comment:
I haven't had the opportunity to debug this particular part of the config, but seems to me the exclude rule is getting lost with the updates.

Are you sure webpack.config.js is meant to replace this block?

Reply:
True, good that you noticed this, basically the webpack.config.js was still acting, but the final "merge" in the main.js export was not good, i copied that from here:  https://storybook.js.org/docs/react/configure/webpack#using-your-existing-config
It seems enough to just move the customisation we make directly in the main.js in this webpackFinal key, instead, so i went for that and removed the webpack.config.js